schema/location.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
looks like our schema updating script is broken - I ran it to see what else has changed and:

* the schema/metadata.json script is missing the git information
* schema/config.json is missing the new logging bits

Neither matter here, but something we should tidy away later: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4230